src/nwbinspector/_nwb_inspection.py

```diff
@@ -126,7 +126,9 @@ def inspect_all(
     if progress_bar_options is None:
         progress_bar_options = dict(position=0, leave=False)
 
-    if in_path.is_dir():
+    if in_path.is_dir() and (in_path.match("*.nwb*")) and (in_path / ".zgroup").exists():
```

Comment:
I'm not sure if `(in_path / ".zgroup").exists()` is the appropriate check here for a Zarr NWB file or if it is even necessary

Reply:
Can we use https://github.com/hdmf-dev/hdmf-zarr/blob/dev/src/hdmf_zarr/backend.py#L78 ?